### ISSUE TITLE
Debugger code completion does not return any items

### ISSUE THREAD

**Issue by mrward** [mono/monodevelop#7412]
Code completion in the Debugger Immediate Pad and the Watch window no longer works in VS Mac 8.1.0.2076.

Build Information
Release ID: 801002084
Git revision: 83a71a45834f8deef178762c25f33177df5a74c0
Build date: 2019-04-19 16:55:23+00
Build branch: master
Xamarin extensions: ee1e3354fcbc9649f6c247f444dd4a4f2aa51a13

To repro:

1. Create a C# console project.
2. Put a breakpoint on the writeline.
3. In the immediate window/watch window try to get code completion for args.

IDE log:

```
 An unhandled exception has occurred. Terminating Visual Studio? False
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.Debugger.DebuggingService.GetCompletionDataAsync (Mono.Debugging.Client.StackFrame frame, System.String exp, System.Threading.CancellationToken token) [0x0000a] in /Users/vsts/agent/2.150.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/DebuggingService.cs:1465 
  at MonoDevelop.Debugger.DebuggerConsoleView.PopupCompletion () [0x000a5] in /Users/vsts/agent/2.150.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/DebuggerConsoleView.cs:100 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-08/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/AsyncMethodBuilder.cs:1023 
  at MonoDevelop.Ide.DispatchService+GtkSynchronizationContext+TimeoutProxy.HandlerInternal (System.IntPtr data) [0x00014] in /Users/vsts/agent/2.150.0/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/DispatchService.cs:77 
```

A quick debug of this and it looks like the document has no associated IDebuggerCompletionProvider.

    var completionProvider = doc.GetContent<IDebuggerCompletionProvider> ();

> VS bug [#853396](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/853396)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-16] Comment by **xamarinc**



Fixed in version 8.1.0.2224 (master-vnext)



Pull Request #629 merged by: Jason Imison  
 Author: xamarin  
 Commit: 35451194f357d7d2203931ed1acdde6df3daf43e (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-20] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #629 merged by: Jason Imison  
 Author: xamarin  
 Commit: 35451194f357d7d2203931ed1acdde6df3daf43e (xamarin/monodevelop)
